lapce-ui/src/tab.rs

```diff
@@ -631,10 +631,6 @@ impl LapceTab {
                 if mouse.button.is_left() && ctx.is_active() {
                     ctx.set_active(false);
                 }
-                if data.drag.is_some() {
```

Comment:
A better way is to check if the drag is a panel here.

Reply:
thanks, I've updated it.

let me know what you think